isabelle/examples/Allocator.thy

```diff
@@ -2,13 +2,13 @@
     Author:     Stephan Merz, Inria Nancy
     Copyright (C) 2009-2022  INRIA and Microsoft Corporation
     License:    BSD
-    Version:    Isabelle2021-1
+    Version:    Isabelle2024
 *)
 
 section \<open>A Simple Resource Allocator\<close>
 
 theory Allocator
-imports Constant
+imports "../Constant"
```

Comment:
@muenchnerkindl, if the theories are referenced from the same ROOT file, then the other theories are imported without the relative path, just `imports Constant`. Can I revert this change as well?

Reply:
Of course. Made the change in interactive mode when I modified both the theories in the main directory and in the examples directory.